setup.json

```diff
@@ -120,7 +120,7 @@
     "dev_precommit": [
       "astroid==1.6.6; python_version<'3.0'",
       "astroid==2.2.5; python_version>='3.0'",
-      "pep8-naming==0.8.2",
+      "pep8-naming==0.4.1",
```

Comment:
why the downgrade?

Reply:
This was necessary because otherwise the pre-commit failed because `prospector` has this requirement. We can tackle these in the other issue about relaxing dependency requirements